docs/source/release_notes/release-notes.rst

```diff
@@ -42,8 +42,11 @@ develop
     *    - Type
          - Change
 
-    *    -
-         -
+    *    - |improved| |devbreak|
+         - Improved performance by preventing excessive reads from the _namespace table when initializing SweepStrategyManager.
```

Comment:
I don't think a test would make much sense here. How would you test that you have fewer round trips? Fewer, compared to what exactly?

The improvement comes from the change TableRemappingKeyValueService.getMetadataForTables(). Note how we used to have a for loop over table references and called tableMapper.mapToFullTableNames every iteration for one table at a time. If a table was a miss (for example, it's not a mapped table, which is true for quite a few tables in our large internal product), it would cause reloading the whole _namespace table again. So if you had N unmapped tables, you'd have N full loads of _namespace.

With cholve's change, we will only load the _namespace once.

Can we please merge this?